soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10587,6 +10563,45 @@ static Vec3f D_80854814 = { 0.0f, 0.0f, 200.0f };
 static f32 D_80854820[] = { 2.0f, 4.0f, 7.0f };
 static f32 D_8085482C[] = { 0.5f, 1.0f, 3.0f };
 
+void Player_AssignTunicBoots(Player* this, PlayState* play) {
```

Comment:
The original method this was in `func_80835F44` was renamed to `Player_UseItems` in decomp. Knowing that, maybe instead of `Assign` here we could use `Use` instead. `Player_UseTunicBoots`, thoughts?

Reply:
Sounds good, only used assign to fit the terminology used by the enhancement name 👍🏼

Reply:
Done